### ISSUE TITLE
How can I get the result from TS

### ISSUE THREAD

**Issue by yizhiai1994** [verenich/time-prediction-benchmark#2]
Dear Verenich:
I have read your paper and I want to try your experiment again, but I didn't find how to get the result from TS in your the project. Did you get the result from Prom5?
                                                                                                                                                Sun

**Comment by verenich** [comment]
Hi there,

TS is a transition system as described in [Aalst, 2011](https://dl.acm.org/citation.cfm?id=1891234). I used the implementation available as a ProM plugin, I just had to modify the source code so that it would output predictions **after each event**, for benchmarking purposes.

**Comment by yizhiai1994** [comment]
Dear Verenich:
Thank you for your answer. I am new to ProM and I have some questions about how to modify the source code. 1. Which version of ProM did you use? (I just found FSM analysis in ProM5.2) 2. How can I modify  the source code? (Could I modify the java file in ProM? or I should use  other methods?)

**Comment by verenich** [comment]
Yes, the logic is based on FSM analyzer in ProM 5.2, you can pull [ProM 5 source code](https://svn.win.tue.nl/repos/prom/Packages/ProM5/Trunk/) and modify it there

**Comment by verenich** [comment]
Or you can use ready implementation from here https://svn.win.tue.nl/repos/prom/Packages/NiekTax/Trunk/src/org/processmining/niektax/qut/CalculateRemainCycleTime.java
